other-docs/guides/upgrading/v7.md

```diff
@@ -0,0 +1,141 @@
+- A [new Robots API](https://make.wordpress.org/core/2021/02/19/robots-api-and-max-image-preview-directive-in-wordpress-5-7/) provides control over the robots meta tag and search engine previews
+- Lazy loading for iframes
+- Ability to send password reset links to users
+
+Be sure to read the [release post](https://wordpress.org/news/2021/03/esperanza/) and in addition the full [WordPress 5.7 Field Guide can be found here](https://make.wordpress.org/core/2021/02/19/robots-api-and-max-image-preview-directive-in-wordpress-5-7/).
+
+### Experience Block Insights
+
+Continuing from previous improvements to tracking the performance of Experience Blocks in v6, Altis v7 adds a dedicated dashboard for showing all Experience Blocks across the site with options for sorting by views and conversion rate.
+
+When editing blocks warnings are shown to authors if their selected conversion goals can not be tracked, and if they are missing fallback content.
+
+In addition there is a new, more detailed overview of Experience Block data that more clearly shows audience coverage, views and conversion rate changes over time and also detailed breakdowns of how each variant is performing compared to the fallback.
+
+On the software level several new REST API endpoints have been added for fetching blocks and block anlaytics data, along with a background post type that can be used for storing persistent data.
+
+### Authorship
+
+Altis now supports adding multiple authors to posts as well as managing guest authors, either as full or partial users.
+
+This feature was funded by and built in collaboration with Siemens.
+
+### Better Reusable Blocks
+
+In addition to the WordPress core improvements to Reusable Blocks Altis v7 bundles several improvements to the Resuable Blocks interface. These include:
+
+- Ability to search by block title within the editor
+- Usage count
+- Links to posts using a block
+- Resuable Block categories
+
+
+## Other Features and APIs
+
+### Local Server
+
+- Added a `mysql:` link to the output from `composer server db info` for software that supports the format. This allows using command or control click in the terminal to open the link.
+- A new `composer server db exec` command allows executing SQL from the command line without logging into the container or using some other tool
+- Experimental support for [Mutagen.io](https://mutagen.io) volume sharing. This can improve speed and file synchronisation issues on MacOS and Windows in some cases
+
+### Search
+
+- User and terms indexables can now be turned on or off via config
+- Better handling for large requests such as documents during indexing
+- Improved default search algorithm with a tighter scope for matches
+
+### Other Changes
+
+Altis v7 includes many other small changes, consult the changelog for full details of all the changes shipped in this release.
```

Comment:
Can we link the full changeling?

Reply:
It doesn't exist yet as we haven't done the release so it would have to be added later